lib/Net/Twitter/Lite/WithAPIv1_1.pm

```diff
@@ -20,21 +20,11 @@ sub new {
     my $class = shift;
     my %options = @_;
 
-    # Twitter now requires SSL connections. Since Net::Twitter::Lite is used
+    # Twitter now requires SSL connections. Since Net::Twitter::Lite was used
```

Comment:
I think the present tense, is, is still appropriate here. It may still be used for compatible services that don't requires SSL. I have no way of knowing, for sure, but that's why we went through a deprecation cycle and why `ssl => 0` is still an option.